BrowserKit/Sources/SiteImageView/URLProcessing/NetworkRequest.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import Foundation
+
+protocol NetworkRequest {
+    func fetchDataForURL(_ url: URL, completion: @escaping ((Result<Data, SiteImageError>) -> Void))
+}
+
+class HTMLDataRequest: NetworkRequest {
+
+    enum RequestConstants {
+        static let timeout: TimeInterval = 5
+        static let userAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15"
```

Comment:
I'm just copying the desktop UA directly here but in future I would like to have a place we can pull shared resources from that lives inside BrowserKit. Many of the things that currently live in Shared could probably live there too.